coding/file_reader.cpp

```diff
@@ -14,9 +14,9 @@
 #endif
 
 // static
-uint32_t const FileReader::kDefaultLogPageSize = 10;
+uint32_t const FileReader::kDefaultLogPageSize = 10; // page size is 2^10 = 1024 = 1kb
```

Comment:
nit: Start sentences with a capital letter and end them with a dot for better readability.